### ISSUE TITLE
Long texts render very slowly (lot of time spent in render and painting)

### ISSUE THREAD

**Issue by scharf** [ant-design/ant-design#13009]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.10.4

### Environment
chrome 70 and safari 12.0

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/03rxw2q260)

### Steps to reproduce
- create a `<div>` with 10,000 words of plain text
- rendering takes a few seconds with `antd.css` (without `antd.css` rendering is fast)

### What is expected?
To be as fast as without `antd.css`

### What is actually happening?
The ui is non responsive (for 3-4 seconds) while rendering the page. It takes a lot of time in the browser `Rendering`.

Here are some example screenshots form the for chrome 70 performance monitor on a MacBook Pro  (2017):

![rendering](https://user-images.githubusercontent.com/3731059/48146036-76cdf580-e2b4-11e8-85c4-711e89f0848b.jpg)

---
It works fine in Firefox 63 and  Edge 17.17134.

**Comment by scharf** [comment]
I think I figured out what causes the problem: it is `font-variant: tabular-nums`:
```css
body {
  // ...
  font-variant: tabular-nums;
}
```

If I set it to `normal` then there is no performance problem anymore...

**Comment by scharf** [comment]
I added another sandbox with a toggle for the `font-variant` (to set it to `normal`)

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/945pr99nkp)

**Comment by afc163** [comment]
Right, we found this problem too in https://github.com/ant-design/ant-design/issues/11843#issuecomment-415021817 and patch it for select specificly in https://github.com/ant-design/ant-design/commit/1a30e5de7878eb8e956ed8a7a1cc4df0d747316d

**Comment by scharf** [comment]
...I think we can close this issue

**Comment by cctv1005s** [comment]
@scharf 

how could you find it?

amazing...
